src/Components/ClusterRules/ClusterRules.spec.ct.js

```diff
@@ -186,9 +186,9 @@ describe('cluster rules table', () => {
   });
 
   describe('defaults', () => {
-    it('only first item expanded', () => {
```

Comment:
This is expected that the first item is expanded by default, you can't edit the test like this.

Reply:
Its not expanded anymore, so why I shouldn't change the test

Reply:
Showing the first recommendation expanded has been a business rule and a feature. There was no request to change it even though I see that RHEL Advisor stopped doing it for some reason. We need to keep the behavior as it has been.